run.py

```diff
@@ -14,8 +14,6 @@ def create_app(config_filename: str) -> Flask:
 
     db.init_app(app)
 
-    migrate = Migrate(app, db)
```

Comment:
this was removed because it was not being used?

Reply:
No this should not be removed. @KushalBeniwal change it to
```Migrate(app, db)```
Just remove the variable

Reply:
thank you for checking @devkapilbansal 🙌🏾 I wasn't really sure. Everything else seems fine, I am ready to approve after this little change